android/lib/src/main/java/com/swmansion/gesturehandler/GestureHandler.java

```diff
@@ -30,7 +30,7 @@
   public static final int DIRECTION_UP = 4;
   public static final int DIRECTION_DOWN = 8;
 
-  private static int MAX_POINTERS_COUNT = 11;
```

Comment:
Why it was 11 though? And not 10?

Reply:
I'll @kmagiera, don't really know

Reply:
He doesn't know it either, it's how it is 🤷